lib/aur-sync

```diff
@@ -207,7 +207,7 @@ trap 'trap_exit' EXIT
 
 # Append remaining options to the aur-build command-line
 if (( $# )); then
-    build_extra_args+=("$@")
+    mapfile -t -d' ' build_extra_args < <(grep -oP ".*?\K-- .*" <<<"$@")
```

Comment:
This could be also [shorter](https://github.com/AladW/aurutils/commit/33eaf878ea2ad0b20af25e0708175d07593f631b) but will need a shellcheck mask.